doc/man3/X509_STORE_CTX_new.pod

```diff
@@ -93,13 +103,29 @@ For L<X509_STORE_CTX_verify(3)>, which takes by default the first element of the
 list of untrusted certificates as its verification target,
 this can be also set indirectly using X509_STORE_CTX_set0_untrusted().
 
+X509_STORE_CTX_init_rpk() sets up I<ctx> for a subsequent verification
+operation for the I<target> raw public key.
+It behaves similarly to X509_STORE_CTX_init().
+The I<target> raw public key can aslo be supplied separately, via
+X509_STORE_CTX_set0_rpk().
+The I<target> public key is not copied (its reference count is not updated),
+and the caller must not free it before verification is complete.
```

Comment:
Ah! Right....answers my earlier question. I'm wondering why we have made this decision, rather than just upping the ref count.

Reply:
I believe it's the same case as with an `X509`